### DISCUSSION TITLE
Movie Downloading

### DISCUSSION THREAD

**Discussion by DinoVikingo** [anidl/multi-downloader-nx#155] [Q&A]
Hi!
I want to download the Overlord OVA's but they're listed as movies and i really don't know how to write the command haha, could you help me please? :,D

I tried with: `aniDL.exe --service 'funi' -s 103522 -q 0 --subLang esLA -e m0001 --dub jaJP`
![image](https://user-images.githubusercontent.com/89960654/140823204-cf508472-2d05-43ef-8fbe-12709c65bac5.png)

But i'm pretty sure that's not the correct form

**Comment by izu-co** [comment]
Try -e M1,M2 or -e M1-2
